osu.Game/Overlays/VolumeOverlay.cs

```diff
@@ -130,10 +140,22 @@ public bool Adjust(GlobalAction action, float amount = 1, bool isPrecise = false
             return false;
         }
 
+        private void volumeMeterSelectionChanged(VolumeMeter meter, bool isSelected)
+        {
+            if (!isSelected)
+                volumeMeters.Deselect();
+            else
+                volumeMeters.Select(meter);
```

Comment:
I'm so confused. Why does this need to exist if you've made a container to manage selection logic?